pkg/devserver/devserver.go

```diff
@@ -372,7 +375,7 @@ func createInmemoryRedis(ctx context.Context, tick time.Duration) (rueidis.Clien
 	// If tick is lower than 250ms, tick every 100ms.  This lets us save
 	// CPU for standard dev-server testing.
 	poll := time.Second
-	if tick < defaultTick {
+	if tick < DefaultTick {
 		poll = time.Millisecond * 50
```

Comment:
Suggesting that we update the comment from "100ms" to match that this is "50ms" (or update to 100ms) - either way works.